aries-site/src/components/seo/Meta.js

```diff
@@ -20,7 +20,7 @@ export const Meta = ({
   const csp = `default-src 'self' 'unsafe-eval'; 
   style-src 'self' *.hpe.com/hfws-static/5/css/ 'unsafe-inline';
   connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://www.google-analytics.com https://www.github.com/grommet/ https://eyes.applitools.com *.hpe.com/hpe/api/;
-  media-src 'self' https://d3hq6blov2iije.cloudfront.net/media/HPE+Design+System-v3.mp4;
+  media-src 'self' https://d3hq6blov2iije.cloudfront.net/media/HPE+Design+System-v3.mp4 https://d3hq6blov2iije.cloudfront.net/media/BILL-ACCESS-VID.mp4;
```

Comment:
Does this achieve the same effect?

```suggestion
  media-src 'self' https://d3hq6blov2iije.cloudfront.net/media/;
```